hw_2/src/main/java/com/epam/spring/homework2/beans/OtherBean.java

```diff
@@ -0,0 +1,23 @@
+package com.epam.spring.homework2.beans;
+
+import org.springframework.beans.BeansException;
+import org.springframework.beans.factory.config.BeanFactoryPostProcessor;
+import org.springframework.beans.factory.config.ConfigurableListableBeanFactory;
+import org.springframework.stereotype.Component;
+
+@Component
+public class OtherBean implements BeanFactoryPostProcessor {
```

Comment:
same remark regarding naming(it should end with `BeanFactoryPostProcessor`),
 please update name to smth like `ReplaceInitMethodBeanFactoryPostProcessor`

Reply:
fixed